06-find.md

```diff
@@ -426,6 +448,9 @@ $ grep FE $(find .. -name '*.pdb')
 > 3. `grep -w of haiku.txt`
 > 4. `grep -i of haiku.txt`
 
+> Using the grep function and the options we have learned so far, 
```

Comment:
This challenge seems repetitive, this exact problem was solved step-by-step above. Modify or remove?

Other than that I like the breakup of the options and building up of the search pipeline, a great addition!